bioimageio/spec/__init__.py

```diff
@@ -16,6 +20,6 @@
 save_raw_node = utils.IO.save_raw_node
 serialize_raw_node_to_dict = utils.IO.serialize_raw_node_to_dict
 
-__version__ = _get_args(ModelFormatVersion)[-1]
-
 # autogen: stop
+
+__version__ = yaml.load(pathlib.Path(__file__).parent / "VERSION")["version"]
```

Comment:
It's a JSON file, so we should use `json.load`, or would `yaml.load` also work?

Reply:
as it happens to also be valid yaml syntax `yaml.load` also works... But if it fundamentally is a json I'll better change it already...

Reply:
Interesting, so yaml is a super set of json? didn't know you can load it with yaml.

Reply:
almost. there are some differences.. https://stackoverflow.com/a/21585224